src/UI/Art/SpriteArtist/UnitArtist.java

```diff
@@ -36,12 +38,13 @@ public void drawUnitIcons(Graphics g, Unit unit, double x, double y, int animInd
    */
   public void drawUnit(Graphics g, Unit unit, int x, int y, int animIndex)
   {
-    // Convert "real" game-model location to a draw-space location.
     int drawX = (int) (myView.getTileSize() * x);
     int drawY = (int) (myView.getTileSize() * y);
+    boolean tired = unit.isStunned || (unit.isTurnOver && unit.CO == myGame.activeCO);
+    AnimState state = (tired) ? AnimState.TIRED : AnimState.IDLE;
```

Comment:
This seems duplicative-yet-different of `SimpleMoveAnimation.getAnimState()`.

Reply:
I pulled it up out of the `UnitSpriteSet` drawing function, since it no longer needs to know the unit's state when the caller can tell it exactly what to draw.

...and it's not really duplicated logic, they're different functions using the same interface :P